src/Transition/CreateNewIDFUsingRulesV25_2_0.f90

```diff
@@ -382,6 +382,12 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 !                 CurArgs = CurArgs + 1
 
               ! If your original object starts with A, insert the rules here
+              CASE('AirLoopHVAC:UnitaryHeatPump:AirToAir:MultiSpeed')
+                 CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                 nodiff=.false.
+                 OutArgs(1:11)=InArgs(1:11)
+                 OutArgs(12) = " "
+                 OutArgs(13:CurArgs)=InArgs(13:CurArgs)
```

Comment:
I've tried several things on the right side of OutArgs(12) to get this to work but it continues to result in the old numeric value posing for the new numeric field.

<img width="796" height="244" alt="image" src="https://github.com/user-attachments/assets/63741cbd-f4dd-44f7-b8c6-b9369a04ad87" />

If you look at how I changed the existing test files I am trying to replace an old deprecated numeric field with a new numeric field having a blank line. The old Minimum Outdoor Dry-Bulb Temperature for Compressor Operation field is removed and replaced with DX Heating Coil Sizing Ratio.

    -8.0,                    !- Minimum Outdoor Dry-Bulb Temperature for Compressor Operation {C}
    ,                        !- DX Heating Coil Sizing Ratio